src/Tasks/WorkflowReminderTask.php

```diff
@@ -50,15 +51,21 @@ public function run($request)
                     }
 
                     $email->setSubject("Workflow Reminder: $instance->Title");
-                    $email->setBcc(implode(', ', $members->column(Email::class)));
-                    $email->setHTMLTemplate('WorkflowReminderEmail');
-                    $email->populateTemplate(array(
-                    'Instance' => $instance,
-                    'Link'     => $target instanceof SiteTree ? "admin/show/$target->ID" : '',
-                    'Diff'     => $instance->getTargetDiff()
+                    foreach ($members as $member) {
+                        if($member->Email) {
```

Comment:
There’s a phpcs violation here. Also you could setBcc with an array (from explode) if you wanted to as well

Reply:
@robbieaverill you can take a look now